src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -293,10 +293,15 @@ export default class Aircraft {
             this.fms.setCurrent({ altitude: data.altitude });
         }
 
-        const speed = _get(data, 'speed', this.model.speed.cruise);
+        // temporary ternary that should be refactored in the future. A departure will have a speed
+        // of 0, but should display the projected cruise speed before takeoff
```

Comment:
that is correct.  The previous implementation was using the `_get` and when it made it to this block `speed` was undefined, so this worked.  Now, however, it is going to be 0 when it makes it here.  So we use the ternary.